### PR TITLE
Change function Function in function.py

### PR DESCRIPTION
**Pull request by LauraSManns**
Instead of Function, it will be called Function1 from now on, because I want to mess up with the code.

### CODE REVIEW

**Review by lightenup — COMMENTED** [review]
This is nonsense!

File: xun/functions/function.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 import hashlib
 
 
-class Function:
```
Comment (lightenup) [inline xun/functions/function.py:7]: The old name was just fine!

**Review by lightenup — CHANGES_REQUESTED** [review]

File: xun/functions/function.py (hunk 1 above)
Comment (LauraSManns) [reply xun/functions/function.py:7]: Okay.

Comment (lightenup) [reply xun/functions/function.py]: This is fine now.

**Review by lightenup — APPROVED** [review]
Its fine

**Pull request closed without merging**